templates/system/page.tpl.php

```diff
@@ -166,14 +166,13 @@
     </nav>
   <?php endif; ?>
     
-  <?php if (theme_get_setting('ec_europa_improved_website', 'europa')): ?>
-    <?php if (!$is_front || $is_front && theme_get_setting('ec_europa_improved_website_home', 'europa')): ?>
+  <?php if ($settings['improved'] && (!$is_front || $is_front && $settings['identification_home'])): ?>
     <div class="container-fluid page-header__site-identification">
       <h3><?php print $site_name; ?></h3>
     </div>
-    <?php endif; ?>
   <?php endif; ?>
-    
+
+  <?php if (!$settings['improved'] || ($settings['improved'] && $settings['improved_header'] == 'complete') && (!empty($page['custom_title']) || !empty($title))): ?>
```

Comment:
Please change
`$settings['improved_header'] == 'complete'`
to
`$settings['improved_header'] === 'complete'`